src/main/java/com/learntechfree/ltf/article/ArticleResolver.java

```diff
@@ -8,10 +9,12 @@
 
 @Component
 public class ArticleResolver implements GraphQLResolver<Article> {
+    
     @Autowired
     private ArticleService articleService;
 
-    public Optional<Article> getArticle(Long id){
+    public Optional<Article> getArticle(Long id) {
```

Comment:
Here, lets handle the necessary exception in `ArticleService` and avoid the `Optional`  type which is unsure about the data.